x-pack/platform/plugins/shared/ml/public/application/memory_usage/nodes_overview/memory_preview_chart.tsx

```diff
@@ -123,8 +123,8 @@ export const MemoryPreviewChart: FC<MemoryPreviewChartProps> = ({ memoryOverview
         }
       />
       <Settings
-        theme={{ chartMargins: LEGACY_LIGHT_THEME.chartMargins }}
-        baseTheme={chartsService.theme.useChartsBaseTheme()}
+        theme={{ chartMargins: LIGHT_THEME.chartMargins }}
```

Comment:
I think we can remove all of these. At some point the `LEGACY` theme had margins with all `10`. But this is no longer the case so this is not needed.

```suggestion
```

Reply:
So we have `LEGACY_CHART_MARGINS` that are all 10px and instead DEFAULT_CHART_MARGINS (used in borealis) all 0.

Reply:
Oh yeah sorry, some how when I looked at this last week I saw even `LEGACY_LIGHT_THEME.chartMargins` were all `0` but I am mistaken 🙃. At any rate it would be nice to remove this in general and align on a consistent chart styling to where this is not necessary.